apps/frontend/app/admin/contest/[contestId]/_components/ContestOverallTabs.tsx

```diff
@@ -62,6 +63,15 @@ export default function ContestOverallTabs({
     skip: !contestId
   })
 
+  const formatScore = (score: number): string => {
+    const fixedScore = Math.floor(score * 1000) / 1000
+    return fixedScore % 1 === 0
+      ? fixedScore.toFixed(0)
+      : Math.floor(fixedScore * 10) % 10 === 0
+        ? (Math.floor(fixedScore * 10) / 10).toFixed(1)
+        : fixedScore.toString()
+  }
+
```

Comment:
```suggestion
  const formatScore = (score: number): string => {
    const fixedScore = Math.floor(score * 1000) / 1000
    return fixedScore.toString()
  }
```

이렇게 작성해도 똑같이 작동할 것 같은데 확인해주실 수 있나요?

추가로, 중첩된 삼항 연산자를 써야 하는 경우,
조건문으로 바꿔서 작성하면 좋을 것 같아요!

Reply:
똑같이 작동하는거 같습니다!!!! 최고..